ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -77,15 +78,20 @@ class GameOptionsViewController: UIViewController {
 
     // Make an api call to create a game room, and then go to CountdownTimerViewController if it succeeds.
     @IBAction func onStartGameClicked(_ sender: Any) {
-        print("game started clicked")
         var newGame = ["max_num_of_questions" : numOfQuestions ?? 0,
+                       "id" : FirebaseClient.instance.createGameRoomId().key,
                    "current_question" : 0,
                    "is_public" : isPublic ?? true,
                    "max_num_people" : numOfPlayers ?? 5,
-                   "name" : "test",
+                   "name" : self.selectedCategory ?? "",
```

Comment:
I was expecting the name to be something the user inputs that let's other players know what game it is.  Like a chatroom name.